drivers/wifi/nrf700x/zephyr/src/qspi/inc/ficr_prog.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @brief Header containing common functions for RPU hardware interaction
+ * using QSPI and SPI that can be invoked by shell or the driver.
```

Comment:
Need to modify the text, this is from rpu_fw_if.c

Reply:
done